.github/workflows/build-timestamped-master.yml

```
@@ -13,8 +13,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Set up JDK 11
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v2
         with:
+          distribution: 'adopt'
           java-version: 11
```

The guideline had single quotes around the version. Don't know whether it is also working this way. But I think it is better to use quotes. Check other places as well.

```suggestion
          java-version: `11`
```